alf/algorithms/hypernetwork_algorithm_test.py

```diff
@@ -54,13 +55,14 @@ def assertArrayGreater(self, x, y, eps):
         self.assertEqual(x.shape, y.shape)
         self.assertGreater(float(torch.min(x - y)), eps)
 
-    @parameterized.parameters(('gfsf'), ('svgd2'), ('svgd3'), ('minmax'),
+    @parameterized.parameters(('gfsf', False), ('svgd2', False),
+                              ('svgd3', False), ('minmax', False),
```

Comment:
tested. 
My comment here is that I could just test this function with random inputs, instead of actual MNIST images, since I don't care about "passing the test" in the usual way, I just want eval_uncertainty to run.

Reply:
Agreed, just make sure it runs.

Reply:
will update